### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by ps-amaureder**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+/Dockerfile export-ignore
+/Gemfile export-ignore
+/Gemfile.lock export-ignore
+/phpcs.xml export-ignore
+/phpmd.xml export-ignore
+/Rakefile export-ignore
+/.bin export-ignore
+/ops export-ignore
+/phpmd export-ignore
+/Test export-ignore
```
Comment (tomazpu) [inline .gitattributes]: I think in the repo its called /Tests

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+/WirecardOxidRuleset export-ignore
+
```
Comment (tomazpu) [inline .gitattributes]: i think .bundle is missing

File: .gitattributes (hunk 1 above)
Comment (ps-cgrach) [reply .gitattributes]: Yes.

File: .gitattributes (hunk 2 above)
Comment (ps-cgrach) [reply .gitattributes]: Added

**Pull request merged**